app/models/user.rb

```diff
@@ -2,8 +2,9 @@
 # Fields are all devise settings; most of the methods relate to call list mgmt.
 class User
   include Mongoid::Document
-  include Mongoid::Enum
   include Mongoid::Userstamp::User
+  extend Enumerize
+  extend ActiveModel::Naming
```

Comment:
This is only for feedback (we do this at Sonatype, I find it extremely useful), is when you make the change, comment on the thread where it originated with the commit sha so that the reviewer can find your change (for one), and for traceability in the future. We do something like:

In: e561096